### PR TITLE
Add pci address element and attributes to serial device

### PR DESCRIPTION
**Pull request by jinqi7**
Signed-off-by: qijing <jinqi@redhat.com>

### CODE REVIEW

File: virttest/libvirt_xml/devices/serial.py [hunk 1]
```diff
@@ -23,11 +24,19 @@ def __init__(self, type_name='pty', virsh_instance=base.virsh):
...
         accessors.XMLAttribute('target_model', self, parent_xpath='/target',
                                tag_name='model', attribute='name')
-        accessors.XMLElementDict('alias', self, parent_xpath='/',
-                                 tag_name='alias')
         accessors.XMLElementList('sources', self, parent_xpath='/',
                                  marshal_from=self.marshal_from_sources,
                                  marshal_to=self.marshal_to_sources)
+        accessors.XMLElementDict('address', self, parent_xpath='/',
+                                 tag_name='address')
+        accessors.XMLAttribute('address_type', self, parent_xpath='/',
```
Comment (chunfuwen) [inline virttest/libvirt_xml/devices/serial.py]: I assume those attributes configuration are not needed, just accessors.XMLElementDict is enough

Comment (jinqi7) [reply virttest/libvirt_xml/devices/serial.py]: Changed to use XMElementNest with Address class instead of XMLElementDict.

Comment (jinqi7) [reply virttest/libvirt_xml/devices/serial.py]: Changed to use ElementDict only for no special needed for sub attribute.

**Comment by yanjing1993** [comment]
Suggest to reserve 'alias' for serial device, I think it is necessary to test it.

**Comment by jinqi7** [comment]
> Suggest to reserve 'alias' for serial device, I think it is necessary to test it.

I think it can be added when it's needed.

**Comment by yanjing1993** [comment]
> > Suggest to reserve 'alias' for serial device, I think it is necessary to test it.
> 
> I think it can be added when it's needed.

Reserving alias has no conflict with adding pci address in 'Serial' class, please reserve it.

**Pull request merged**